study-builder/studies/atcp/study.conf

```diff
@@ -1239,6 +1239,180 @@
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedPhysicianInfo"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "MEDICAL_HISTORY",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedMedicalHistory"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENOME_STUDY",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedGenomeStudyShippingInfo"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 2
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "REVIEW_AND_SUBMISSION",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_STATUS",
+        "status": "SubmittedEnrollment"
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    }
+    #REGISTRATION_TYPE events for DSM
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "REGISTRATION",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_TYPE",
+        "status": "Self"
+      },
+      "preconditionExpr": """operator.studies["atcp"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("DIAGNOSED")"""
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "REGISTRATION",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_TYPE",
+        "status": "PortalUser"
+      },
+      "preconditionExpr": """operator.studies["atcp"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("CHILD_DIAGNOSED")"""
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    },
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "REGISTRATION",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
+        "workflow": "REGISTRATION_TYPE",
+        "status": "Dependent"
+      },
+      "preconditionExpr": """operator.studies["atcp"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("CHILD_DIAGNOSED")"""
```

Comment:
This is the same expression as the `PortalUser` one above?

Reply:
Expr is same but trigger is after REGISTRATION COMPLETE